app/topics/content/content.component.ts

```diff
@@ -0,0 +1,224 @@
+import { Component, OnInit, NgZone } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { ToasterService } from 'angular2-toaster';
+
+import 'docsApi';
+
+import { UserService } from '../../core/user/user.service';
+import { User } from '../../core/user/user.model';
+import { OOApiService } from '../../core/api/oo-api.service';
+import { CONFIG } from '../../config.constant';
+
+declare var DocsAPI: any;
+
+declare var window: {
+  angularComponentRef: any;
+};
+
+function onRequestHistory() {
+  window.angularComponentRef.zone.run(() => {
+      window.angularComponentRef.onRequestHistory();
+    }
+  )
+}
+
+function onRequestChangeHistoryData(data: any) {
+  console.log(data);
+  window.angularComponentRef.zone.run(() => {
+      window.angularComponentRef.onRequestChangeHistoryData(data);
+    }
+  )
+}
+
+@Component({
+  selector: 'hip-content',
+  template: `
+  <div class="form" style="margin: 0;padding: 0;height: 100%;">
+    <div id="iframeEditor">
+    </div>
+  </div>
+`
+})
+export class ContentComponent implements OnInit {
+  private currentUser: User;
+  private topicId: number;
+
+  apiUrl = '';
+  public config = {
```

Comment:
not sure if it is necessary to mock the whole config object, but isn't much of an issue